template.rb

```diff
@@ -88,6 +97,154 @@
 # Remove test folder
 run 'rm -rf test/'
 
+# Add heroku buildpacks for heroku deployment
+file '.buildpacks', <<-FILE
```

Comment:
Is there a way we can move these to actual files to help with maintainability? Have them read in at execution time? Maybe make a folder at the root level to collect all of these?